### DISCUSSION TITLE
coc-rename open changed files in buffer rather than tabs

### DISCUSSION THREAD

**Discussion by ehaugw** [neoclide/coc.nvim#4800] [Q&A] [answered]
I can not figure out how to open the changed files after a coc-rename in buffers in the existing window rather than new tabs. Does anyone how to fix this?

**Comment by fannheyward** [comment, marked as answer]
You can change `workspace.openResourceCommand` to `edit` to open changed files in buffer, by default it's `tab drop`.

Check `:h coc-config-workspace-openResourceCommand`
